less/pages/volunteer.less

```diff
@@ -2,7 +2,7 @@
   .hero-unit-container {
     background: linear-gradient( to right, #4A825D, #406882);
   }
-  .thank-you {
```

Comment:
can't find any `thank-you` class element in the codebase. this is probably some old code that we forgot to remove